web/src/Components/Bridging/BridgeContainer.jsx

```diff
@@ -48,6 +49,13 @@ export default ({ bridges, mode, setMode, bridgeValue, setValue }) => {
         })
   }, [])
 
+  useEffect(() => {
+      if (bridges["Bridge"].length > 0 && sizeFractions.length > 0 ) {
```

Comment:
```suggestion
      if (bridges["Bridge"].length && sizeFractions.length) {
```
May be shortened like this, 0 is false. >0 is True